README.md

```diff
@@ -105,7 +104,31 @@ However, be aware that phantomjs process will get detached (and thus won't exit)
 
 ## `page` object API
 
-  The `page` object that is returned with `#createPage` is a proxy that sends all methods to `phantom`. Most method calls should be identical to PhantomJS API. You must remember that each method returns a `Promise`.
+The `page` object that is returned with `#createPage` is a proxy that sends all methods to `phantom`. Most method calls should be identical to PhantomJS API. You must remember that each method returns a `Promise`.
+
+For list of available methods, properties and events see [PhantomJS documentation of WebPage module](http://phantomjs.org/api/webpage/).
+
+### `page#loadContent`
+
+`page.loadContent` is used to set content and/or URL address of the page.
+
+It differs from `page.invokeMethod('setContent')` by waiting for contents to fully load.
```

Comment:
You mean `page.setContent` right? I don't think they actually use `invokeMethod`